app/src/components/Forms/Form.js

```diff
@@ -35,18 +49,29 @@ class Form extends React.Component {
         render={formikProps => {
           const form = (
             <React.Fragment>
-              {Object.keys(fields).map(id =>
-                React.createElement(fields[id].component, {
-                  id,
-                  label: fields[id].label,
-                  formik: formikProps
-                })
-              )}
+              {formFromFields(fields, formikProps)}
             </React.Fragment>
           );
-
+          const formSections =
```

Comment:
Can you walk me through the formSection is doing here. I see that you are returning children(form, formSections) further down. I'm getting lost in how it's building together.

Reply:
basically --
iterate over each section in the `sections` prop array (e.g. ["sectionOne","sectionTwo"]) and return a formSections object that looks like this:

```
{
  sectionOne: { // all `fields` with section === `sectionOne`}
  sectionTwo: { // all `fields` with section === `sectionTwo`}
}

Reply:
Ok. So we are expecting a consistent naming convention of `sectionNUMBER`? Is it making sure `sectionOne` comes before `sectionTwo`, and `sectionTwo` comes before `sectionThree`?